specification/glossary.md

```diff
@@ -0,0 +1,44 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
+
+Denotes the actual *OpenTelemetry* library in use.  
+See [Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md#library)
+
+<a name="instrumented_library"></a>
+
+## Instrumented Library
+
+Denotes the library instrumented with OpenTelemetry API calls, thus the library
+that is producing telemetry data.
+
+Example: `org.mongodb`.
+
+<a name="instrumenting_library"></a>
+
+## Instrumenting Library
+
+Denotes the library which provides the instrumentation *for* a given library (i.e.
+the [Instrumented Library](#instrumented_library)). Hence, *Instrumented Library*
+and *Instrumenting Library* can be identical, but they don't necessarily have to.
```

Comment:
... can be identical if a library has OpenTelemetry instrumentation built-in / out-of-the-box